linux_os/guide/services/ntp/chronyd_no_chronyc_network/rule.yml

```diff
@@ -20,6 +20,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000096-GPOS-00050
```

Comment:
Same as chronyd_client_only. Does DoD prohibit the use of time servers and clients?

Reply:
This rule includes a setting that prevents Chrony from being remotely controlled.  Since Chrony can act as a client and a server, it is good to limit how it can be connected to.

rationale: |-
    Not exposing the management interface of the chrony daemon on the network diminishes the attack space.